Week2/8_checkTypeOfData.js

```diff
@@ -0,0 +1,29 @@
+let num = 5;
+let str = "name";
+let arr = ["blue", "black", "Red"];
+let boo = false;
+console.log("The value of my variable 'num' is: " + num);
+console.log("num is: Number");
+console.log("The value of my variable 'str' is: " + str);
+console.log("str is: String");
+console.log("The value of my variable 'arr' is: " + arr);
+console.log("arr is: Array");
+console.log("The value of my variable 'boo' is: " + boo);
+console.log("boo is: Boolean");
+
+function compareType() {
```

Comment:
Wow nice function. But I think that in the assignment it is required to `console.log` it every time two variables are not the same.

> Make sure to also show a message when the variables you are comparing are not the same type.

So you could make one function where you pass two variables as `arguments` and then compare those two and if they are not the same, then log that also. And then pass different variables to that function (I can explain it in more detail if you don't fully understand what I'm saying).